src/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Net.Security.Tests
 {
     internal static class TestConfiguration
     {
-        public const int PassingTestTimeoutMilliseconds = 1 * 60 * 1000;
+        public const int PassingTestTimeoutMilliseconds = 4 * 60 * 1000;
```

Comment:
If the server doesn't support the requested protocol, it may take awhile (2 minutes secondhand info) so 4 minutes sounds good to at least see if we get a different exception, for example.